app/src/main/java/org/oppia/app/shim/IntentFactoryShimInterface.kt

```diff
@@ -0,0 +1,19 @@
+package org.oppia.app.shim
+
+import android.content.Context
+import android.content.Intent
+import androidx.fragment.app.FragmentActivity
+
+interface IntentFactoryShimInterface {
```

Comment:
Prefer not calling anything 'interface'. Some naming conventions in codebases follow that style (or prefix the class name with 'I'), but we don't. Instead, name the interface based on the component & implementations with 'impl', e.g.:
- ``IntentFactoryShim``
- ``IntentFactoryShimImpl``

Ditto elsewhere.

Reply:
WIll do :)